lms/envs/common.py

```diff
@@ -4740,6 +4741,7 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
         ENTERPRISE_FULFILLMENT_OPERATOR_ROLE,
         ENTERPRISE_SSO_ORCHESTRATOR_OPERATOR_ROLE,
     ],
+    SYSTEM_ENTERPRISE_PROVISIONING_ADMIN_ROLE: [SYSTEM_ENTERPRISE_PROVISIONING_ADMIN_ROLE]
```

Comment:
what's the purpose of this mapping? Isn't `SYSTEM_ENTERPRISE_PROVISIONING_ADMIN_ROLE ` a system role rather than a feature role?

Reply:
If i remove it; it returns false 👇 
https://github.com/openedx/edx-enterprise/pull/2192/files#diff-20d68ed60a96a5c426f31dce238fe01856337b778a5c60f4e5017e85f1ff5ea5R54